### ISSUE TITLE
React-Bootstrap components are not meeting ADA website accessibility standards

### ISSUE THREAD

**Issue by medicalspanw** [react-bootstrap/react-bootstrap#4371]
## Is your feature request related to a problem? Please describe

Navbar, dropdowns and possibly other components are not compling with ADA website accessibility standards. That is, one cannot use a simple keyboard to navigate through the website when constructed with these components. Using these components requires a touch screen, mouse and/or some tactile interface. People with limited dexterity cannot use these more advanced interfaces and will have a difficult time navigating through a website constructed with these components.

To replicate the problem:
Go to the github users guide for Navbars at: 
https://react-bootstrap.netlify.com/components/navbar/#navbars-overview

Mouse click the "#" symbol that appears after the heading "Overview". Now attempt to navigate through this page using the tab key. 
Click 1: advances to the link "containers". 
This is correct.

Click 2: advances to the Navbar.Brand component labeled "React-Bootstrap". 
This is correct.

Click 3: advances to the search window. 
This is incorrect, as it bypasses the Nav.Links to "Home" and "Link" and bypasses the Nav.Dropdown component labeled dropdown.

## Describe the solution you'd like
The Nav.Links and Dropdowns and other components require something equivalent to an anchor tag for the browser to recognize a landing spot for each tab click. Further, the dropdowns need to open with a return. The dropdown items need to be accessible via the arrow keys and need to be activated with a return key.

(See above)

## Describe alternatives you've considered
The alternative is to use a tactile interface component, such as a mouse. However, this defeats the intent of ADA website accessibility standards.

I have tried adding a discrete anchor tag within these components. This has not worked.

**Comment by jquense** [comment]
Hey thanks for the report, The behavior you're seeing with nav links is a bug we are aware of and working on. You can use `beta.10`, which is what the host docs point too and see that that the nav link behavior is correct. https://react-bootstrap.github.io/components/navbar/#navbars-overview

**Comment by medicalspanw** [comment]
Thanks for the reply.
You are correct. The react-bootstrap.github.io page works as expected.

I am having issues with my implementation. I am using react-boostrap
version: 1.0.0-beta.12. The local version (npm run start) and deployed
version (npm run build) do not allow tabbing to the Nav.Link and
NavDropdown components.

**Comment by jquense** [comment]
yup will be fixed in the next release!

**Comment by medicalspanw** [comment]
Thanks for responding. It is appreciated that you are on top of this issue.
Thanks.
